docs/production/authentication-methods.md

```diff
@@ -588,12 +588,12 @@ domain for your server).
 1. Create a [Sign in with Apple private key][apple-create-private-key].
 
 1. Store the resulting private key at
-   `/etc/zulip/apple/zulip-private-key.key`.  Be sure to set
+   `/etc/zulip/apple-auth-key.p8`.  Be sure to set
    permissions correctly:
 
    ```
-   chown -R zulip:zulip /etc/zulip/apple/
-   chmod 640 /etc/zulip/apple/zulip-private-key.key
+   chown -R zulip:zulip /etc/zulip/
```

Comment:
Don't want to do this -- this will affect every file in that directory :slightly_smiling_face: 

Instead drop the `-R` and refer to just the file itself. The directory should already have the right owner.

(Whereas the file might not, because we aren't providing the exact steps for putting it there, and the reader might create it as root. The same thing would have been true of a newly-created directory like `/etc/zulip/apple/`.)